_includes/help/nav_page_bottom.html

```diff
@@ -0,0 +1,33 @@
+<div class="sm-hide bg-blue-lightest">
+  <nav class="bg-blue-lightest px0">
+    <ul class="js-accordion p0 m0 list-reset">
```

Comment:
I believe `list-reset` removes the padding on the list so the `p0` can be removed.